report_viewer/apps/viewer/tests.py

```diff
@@ -14,9 +14,13 @@
 from django.http import HttpResponse
 from django.template import loader, Template
 from django.urls import reverse
+from django.test.client import RequestFactory
```

Comment:
There is no need for this as `django-pytest` comes with a request factory fixture (`rf`).

This is used in existing tests in this module such as `test_report_metric_middleware_string_to_hash`.